spec/forms/support_interface/country_form_spec.rb

```diff
@@ -192,4 +192,48 @@
       end
     end
   end
+
+  describe "#for_existing_country" do
+    subject(:form) { described_class.for_existing_country(country) }
+
+    let(:country) { create(:country) }
+
+    it "returns a CountryForm" do
+      expect(subject).to be_a(described_class)
+      expect(form.eligibility_enabled).to eq(country.eligibility_enabled)
+      expect(form.eligibility_route).to eq("standard")
+      expect(form.has_regions).to eq(country.regions.count > 1)
+      expect(form.region_names).to eq(country.regions.pluck(:name).join("\n"))
```

Comment:
Can you be explicit here?
Right now you have no regions for example on the country. Why don't you by default create 2/3 regions for the country with specific names and then check that the region_names is correct based on that.